sqlglot/dialects/dialect.py

```diff
@@ -220,6 +220,11 @@ class Dialect(metaclass=_Dialect):
     For example, such columns may be excluded from `SELECT *` queries.
     """
 
+    PUSHDOWN_CTE_ALIAS_COLUMNS = False
+    """
+    Whether or not to qualify outputs before columns.
```

Comment:
I don't think this description is precise enough.
"outputs" is vague, as is "columns".

Maybe we should call this "PREFER_CTE_ALIAS_COLUMN", and the description should describe how the CTE alias columns will override any projection aliases in the subquery.